components/micro-gateway-core/src/main/ballerina/gateway/listeners/api_gateway_listener.bal

```diff
@@ -132,6 +132,12 @@ function initiateGatewayConfigurations(EndpointConfiguration config) {
     initGatewayCaches();
     printDebug(KEY_GW_LISTNER, "Initialized gateway caches");
     initializeAnalytics();
+
+    //Change the httpVersion
+    if (getConfigBooleanValue(HTTP2_INSTANCE_ID, HTTP2_PROPERTY, false)) {
+        config.httpVersion = "2.0";
+        io:println("httpVersion = " + config.httpVersion);
```

Comment:
Lets use the log pakage instead of io package in order to print the logs. And I think we can change this to a debug log